google-marketplace/README.md

```diff
@@ -91,8 +91,7 @@ export TAG=1.3.0 # Datadog Operator version that will be installed
 
 You may also need to customize some parameters (name, namespace, APIKey)
 
-```
-kubectl create ns datadog-agent
+```shell
```

Comment:
would the change made in https://github.com/DataDog/datadog-agent/pull/22643 cover this? 
https://github.com/DataDog/datadog-agent/blob/4035a5ecdaa240632da0475b36726cda5ea1cd2b/google-marketplace/README.md?plain=1#L92-L99